upcloud/load_balancer_test.go

```diff
@@ -99,6 +99,28 @@ func TestMarshalLoadBalancer(t *testing.T) {
 						}
 					]
 				}
+			],
+			"nodes": [
+            	{
+					"networks": [
+						{
```

Comment:
Does this omit `uuid` and `dns_name` on purpose?

Reply:
Those nodes are not that well documented yet and I forgot to update those JSON strings in upcloud package after I noticed that there is also other fields :) 
I'll push update.